lessons/linear-regression/langs/en-us/index.adoc

```diff
@@ -15,165 +15,141 @@
-----
+Just like the `scatter-plot` function, `lr-plot` takes in a Table and the names of *3 Columns*, the first of which will be used to label the points.
+
+[.lesson-instruction]
+- Open your saved Animals Starter File, or @starter-file{animals, make a new copy}.
+- Create an `lr-plot` for the `animals-table`.
+** Use `"name"` for the labels.
+** Use `"age"` for the x-axis.
+** Use `"weeks"` for the y-axis.
 
 @right{@image{images/lr-explained.png, LR explained, 400}}
-`lr-plot` is a function that takes a Table and the names of *3 columns*:
 
-- `ls` -- the name of the column to use for _labels_ (e.g. “names of pets”)
-- `xs` -- the name of the column to use for _x-coordinates_ (e.g. “age of each pet”)
-- `ys` -- the name of the column to use for _y-coordinates_ (e.g. “weeks for each pet to be adopted”)
+. The resulting scatter plot looks like those we’ve seen before!
 
-Our goal is to use values of the variable on our x-axis to _predict_ values of the variable on our y-axis.
+. The @vocab{line of best fit} is now drawn onto the plot.
 
-[.strategy-box, cols="1", grid="none", stripes="none"]
-|===
+. Above the display we see the @vocab{predictor function} for that line.
 
-|
-@span{.title}{Pedagogical Note}
+**  The predictor function is written in slope-intercept form.
+(@math{y = mx + b}, where @math{m} describes the @vocab{slope} or @vocab{rate of change} and @math{b} identifies the @vocab{y-intercept}.)
 
-We prefer the words “explanatory” and “response” in our curriculum, because in other contexts the words “dependent” and “independent” refer to whether or not the variables are related at all, as opposed to what role each plays in the relationship.
-|===
+** In this plot, we can see that the slope of the line is `0.792`, which means that on average, each extra year of age results in an extra @math{0.792} weeks of waiting to be adopted (about 5 or 6 extra days).
 
-[.lesson-instruction]
-- Open your saved Animals Starter File, or @starter-file{animals, make a new copy}.
-- Create an `lr-plot` for the `animals-table`.
-** Use `"names"` for the labels.
-** Use `"age"` for the x-axis.
-** Use `"weeks"` for the y-axis.
+** The y-intercept is `2.285`. This is where the best-fitting line crosses the y-axis.  We want to be careful not to interpret this too literally, and say that a newborn animal would be adopted in 2.285 weeks, because none of the animals in our dataset was that young. Still, the @vocab{regression line} (or @vocab{line of best fit}) suggests that a baby animal, whose age is close to 0, would take only about 3 weeks to be adopted.
+
+** By subsituting an animal's age for _x_ in the predictor function, we can make a _prediction_ about how many weeks it will take to be adopted.
+
+** For example, we predict a 4-year-old animal to be adopted in @math{0.792(3) + 2.285 = 4.665} weeks. That’s the y-value when @math{ x = 3} for a point that falls exactly on the regression line.
+
+. We also see the @math{r-value} is `+0.442`.
```

Comment:
The way mathjax is rendering this is visually jarring. Recommend we switch to plain text, or perhaps use `@math{r}-value` instead

Reply:
@schanzer Funny. I switched them to this cause I didn't like the way the other formatting looked, but I've switched them back.